### DISCUSSION TITLE
[Feature] Support for identifying iOS 'spatial' video

### DISCUSSION THREAD

**Discussion by othyn** [immich-app/immich#11564] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Immich currently supports the sync of spatial video, as presumably its just extra meta data in the standard video container (have no idea how it works), so its all dependant on playback recognition instead of storage.

However, it would be nice for Immich to recognise 'spatial' video from iOS and tag it as such. Just so that it can be filtered on or tagged as. Basically, just a way to identify videos within Immich that are 'spatial'.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by andreymaznyak** [comment]
I'm also interested this feature, and i make researched how we can implement it in immich.
The first, we should use ffmpeg 7.1 and above. 
The second, we should transcode mv-hevc to sbs h264.
And the third we should implement web xr rendering of this videos.

I created example in [separate repository](https://github.com/andreymaznyak/mv-hevc-sandbox)

And i started to [implement web xr for preview 360 photos in web client](https://github.com/immich-app/immich/pull/17988).
